common/iso_config.go

```diff
@@ -81,6 +82,21 @@ func (c *ISOConfig) Prepare(ctx *interpolate.Context) ([]string, []error) {
 							return warnings, errs
 						}
 					case "file":
+						// Windows file handling is all sorts of tricky...
+						if runtime.GOOS == "windows" {
```

Comment:
Is there something in the [std lib](https://golang.org/pkg/path/filepath/) that will help with this? If not, can we make it into a function? I have seen several PRs that do stuff like this so I think there is a need to DRY this up a bit. E.g.

```
func PathFromURL(u) string {
    path := u.Path
    if runtime.GOOS == "windows"
    ...
    return path
}
```